description.md

```diff
@@ -20,6 +20,25 @@ Each topic in this file contains a small description and links to the files cove
 
 ### Drills
 
+Drills are educational activities designed to reinforce learning through repetitive practice.
+The primary goal of drills is to promote mastery and internalisation of skills or knowledge.
+By repeating tasks or exercises, students can reinforce their understanding and enhance their ability to apply learned concepts effectively.
+However, it's important to balance drills with other types of learning activities to maintain student engagement and prevent monotony.
+
+The effectiveness of drills in education depends on various factors, including the type of drill, the subject matter, and the student's learning style.
+
+Some keys considerations when designing drills include:
+1. **Skill acquisition**: By providing repetitive practice, drills help students develop accuracy and speed.
+2. **Reinforcement**: Drills should be used to reinforce skills and knowledge that have already been taught.
+3. **Building foundations**:  By practicing fundamental skills and concepts repeatedly, students can develop a solid base of knowledge, which can support their learning in more complex areas.
+4. **Confidence**: Drills can boost student confidence as they see their progress through repeated practice.
+
```

Comment:
Maybe incrementalism? I mean you start with easy drills and then as you progress they become more complex.

Reply:
I have added it, but naming engagement, rather than incrementalism. Sounds better imo.